app/assets/stylesheets/ama_layout/layout/helper-classes.scss

```diff
@@ -413,10 +421,29 @@ button[disabled],
   font-size: 80%;
 }
 
+.legal{
```

Comment:
in keeping with the rest of the helper class naming conventions, can we change this to be .italic?